shell/platform/common/cpp/text_input_model.h

```diff
@@ -77,9 +96,13 @@ class TextInputModel {
   // Returns true if the cursor could be moved.
   bool MoveCursorToEnd();
 
-  // Get the current text
+  // Gets the current text.
```

Comment:
Related to the above, while you're here could you say that it gets it as UTF-8? While we do generally use UTF-8 when we're using std::string, there's no intrinsic encoding implied by `std::string`, and this API does use other encodings in other methods which could lead to mistakes.